### DISCUSSION TITLE
Custom, dynamic class name bind

### DISCUSSION THREAD

**Discussion by AlexAndrascu** [quasarframework/quasar#14064] [General - Components / Directives / etc] [answered]
I'm trying to add a custom, dynamic class name using v-bind like so:

```
      <div class="column" v-for="card in cards" :key="card.title">
        <div :class="card.class">
```
but not getting the desired output. It works for other binds for example   `<div :classes="card.class">` works fine. 
Any help / pointers would be appreciated. 
The only reference to this I managed to find in the documentation was on #10214 although that wasn't very helpful either.

**Comment by pdanpdan** [comment, marked as answer]
Codepen

**Comment by AlexAndrascu** [comment]


**Reply by pdanpdan** [reply]
Any codepen with Vue where you show what you tried and what is not working.

**Reply by AlexAndrascu** [reply]
Ah. Sorry, I didn't know what you mean. I don't use codepen, I did show above what I've tried and what's not working including what is working :)

not working `<div :class="card.class">` outputs  `<div class>` vs 
working`<div :classes="card.class">` outputs `<div classes="6">`

**Reply by pdanpdan** [reply]
You showed nothing. It's 2 lines of basic Vue html template, with no description of what is in the variables, what is not working, anything.

**Reply by TobyMosque** [reply]
it's still not too late to learn the art of how to use the codepen.

**Reply by AlexAndrascu** [reply]
https://codepen.io/alexandrascu/pen/NWYazdg

**Reply by pdanpdan** [reply]
Html does not allow numeric class names
